recipe/meta.yaml

```diff
@@ -26,14 +26,18 @@ requirements:
   build:
     - toolchain
     - python
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
     - vc 9              # [win and py27]
     - vc 14             # [win and py>=35]
     - boost-cpp {{ version }}
 
   run:
     - python
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```

Comment:
Aren't these suppose to be lower bounded not pinned to a major minor version? 😕

xref: https://github.com/conda-forge/pandas-feedstock/pull/32/files#diff-e178b687b10a71a3348107ae3154e44cR27